rtdb-limit-child-nodes/functions/src/index.ts

```diff
@@ -15,21 +15,39 @@
  */
 
 import * as functions from "firebase-functions";
+import * as admin from "firebase-admin";
 
 import config from "./config";
 import * as logs from "./logs";
 
 logs.init();
 
-export const rtdblimit = functions.handler.database.ref.onCreate(
+// https://stackoverflow.com/questions/37872478/how-to-get-firebase-database-reference-full-path
+function fullPath(ref: admin.database.Reference){
+  return ref.toString().substring(ref.root.toString().length - 1);
+}
+
+function cleanPath(path: string) {
+  const lastIndex = path.length - 1;
+
+  if (path[lastIndex] === "/") {
+    return path.substring(0, lastIndex);
+  }
+
+  return path;
+}
+
+const configPath = cleanPath(config.nodePath);
+
+export const rtdblimit = functions.database.ref(`${configPath}/{nodeId}`).onCreate(
```

Comment:
Please change back to `functions.handler`. We'd like to make sure the extensions emulator commands work correctly